husky_components/toolbar/main.js

```diff
@@ -663,7 +680,43 @@ define(function() {
         },
 
         /**
-         * changes the list items icon and title
+         * Resets the list items icon and title. Tries to set to default, otherwise to null.
+         *
+         * @param listElement
+         */
+        resetMainListItem = function(listElement) {
+            var listItems = this.sandbox.dom.find('span', listElement),
+                itemId = this.sandbox.dom.data(listElement).id,
+                item = this.items[itemId];
+
+            // reset icon
+            this.sandbox.dom.removeClass(listItems.eq(0), '');
+            item.icon = item.defaultIcon;
+            if (!!item.defaultIcon) {
+                this.sandbox.dom.addClass(listItems.eq(0), createIconSupportClass.call(this, item));
+            }
+
+            // reset title
+            item.title = this.sandbox.translate(item.defaultTitle);
+            this.sandbox.dom.html(listItems.eq(1), item.title);
+            this.items[itemId].title = item.title;
+
+            // remove marked class from dropdown-item
+            if (!!item.dropdownItems) {
+                var dropdown = this.sandbox.dom.find(selectors.dropdownMenu + ' li', listElement);
+                this.sandbox.dom.removeClass(dropdown, 'marked');
+            }
+
+            if (this.options.responsive === true) {
+                updateOverflow.call(this);
+            }
+
+            this.sandbox.emit(BUTTON_CHANGED.call(this));
```

Comment:
when calling 'husky.toolbar.item.change' this has been triggered before, yes. 

since i'm still calling the same event, it's more consistent to still trigger it.

what i was also thinking about is, was to specify an event 

`husky.toolbar.item.reset` instead of using the change item. what do you think about that?